404 Not Found - GET https://registry.npmjs.org/jsonschema-tools - Not found

I get this error when attempting to install with the instructions in the README:

```
npm install -g jsonschema-tools
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/jsonschema-tools - Not found
npm ERR! 404
npm ERR! 404  'jsonschema-tools@*' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/README.md
+++ b/README.md
@@ -33,7 +33,7 @@ but with the benifits of decentralization provided by git.
 
 # Usage
 ```
-$ npm install -g jsonschema-tools
+$ npm i -g @wikimedia/jsonschema-tools
 $ jsonschema-tools --help
 
 jsonschema-tools [command]

